Mantle.xcodeproj/project.pbxproj

```diff
@@ -1238,9 +1238,11 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = CD7C6DB71D33AD72002EC294 /* watchOS-Framework.xcconfig */;
 			buildSettings = {
+				BITCODE_GENERATION_MODE = bitcode;
 				CURRENT_PROJECT_VERSION = 1;
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
+				ENABLE_BITCODE = YES;
```

Comment:
The setting is not displayed in Xcode on watchOS targets because bitcode is required, not optional on the platform.